job/job.go

```diff
@@ -0,0 +1,188 @@
+package job
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
```

Comment:
import order is also strange here, usually standard library goes first, than 3rd party packages and then the internal ones (separated by blank line)

Reply:
Yeah, I know.  I was hoping that `go fmt` would sort it but 🤷‍♂ 🤷‍♀

Reply:
oh yeah, strange that your editor didn't sort this out (no pun intended)